apps/hash-graph/libs/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -1086,6 +1088,70 @@ impl<C: AsClient> EntityStore for PostgresStore<C> {
             draft,
         ))
     }
+
+    async fn update_entity_embeddings<A: AuthorizationApi + Send + Sync>(
+        &mut self,
+        actor_id: AccountId,
+        authorization_api: &mut A,
+        embeddings: impl IntoIterator<Item = EntityEmbedding<'_>> + Send,
+    ) -> Result<(), UpdateError> {
+        #[derive(Debug, ToSql)]
+        #[postgres(name = "entity_embeddings")]
+        pub struct EntityEmbeddingsRow<'a> {
+            web_id: OwnedById,
+            entity_uuid: EntityUuid,
+            property: Option<String>,
```

Comment:
Maybe we should call this `property_base_url`? It isn't clear what it is in a few places

Reply:
I'm not sure about it as we also have e.g. `label_property` and also the type suggests that this is a `BaseUrl` (in this specific case it's a string because it's a local definition which matches the Postgres table and as `BaseUrl` does not implement the `ToSql` trait, so we cannot use that here. I expect this to change once we move the type-system crate). In the database, `property` references the `base_urls` tables.

Reply:
Talked about this offline, we're not worrying about this now as `label_property` should then also be renamed to `label_property_base_url`. If this becoming confusing we're going to change it.